pkg/strace/strace_linux.go

```diff
@@ -0,0 +1,16 @@
+// Copyright 2018 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+package strace
+
+import (
+	"golang.org/x/sys/unix"
+	"syscall"
+)
+
+func Wait(pid int) (unix.WaitStatus, error) {
+	var w syscall.WaitStatus
```

Comment:
Can you get rid of `w` because it looks like the call to this function does not care about the return value?

Reply:
They don't care *yet*. let's keep it.

Reply:
but i"m not picky, happy to take it out if you think we should. It's just that this type signature exactly matches the common Go one.